crates/holochain/src/conductor/handle.rs

```diff
@@ -592,19 +608,20 @@ impl<DS: DnaStore + 'static> ConductorHandleT for ConductorHandleImpl<DS> {
     }
 
     async fn deactivate_app(&self, installed_app_id: InstalledAppId) -> ConductorResult<()> {
-        let cell_ids_to_remove = self
-            .conductor
-            .write()
-            .await
-            .deactivate_app_in_db(installed_app_id)
-            .await?;
+        let mut conductor = self.conductor.write().await;
+        let cell_ids_to_remove = conductor.deactivate_app_in_db(installed_app_id).await?;
         // MD: I'm not sure about this. We never add the cells back in after re-activating an app,
```

Comment:
This seems important to answer because hosted happs will get deactivated regularly when broswer tabs close.

Reply:
A good follow-up test will be to take one of the existing conductor tests we have, and do a deactivate and reactivate and see if the test still passes

Reply:
But, to my knowledge, we have not been deactivating apps at all until now.

Reply:
There are now tests to show that deactivation and reactivation works